src/main/webapp/WEB-INF/translate.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
```

Comment:
I'm a little unclear as to why this is in the WEB-INF directory? Also, only Chinese is currently supported, right?

Reply:
Also, javascript and css should be in a separate file.